docs/source/en/glossary.mdx

```diff
@@ -127,12 +144,25 @@ For models employing the function [`apply_chunking_to_forward`], the `chunk_size
 embeddings that are computed in parallel and thus defines the trade-off between memory and time complexity. If
 `chunk_size` is set to 0, no feed forward chunking is done.
 
+## H
+
+### head
+
+The model head refers to the last layer of a neural network that accepts the raw hidden states and projects them onto a different dimension. There is a different model head for each task. For example:
+
+  * [`GPT2ForSequenceClassification`] is a sequence classification head which is a linear layer on top of the base [`GPT2Model`].
+  * [`GPT2LMHeadModel`] is a language modeling head which is a linear layer with weights tied to the input embeddings on top of the base [`GPT2Model`].
```

Comment:
This one is not a good example actually, as `LMHead` is very ambiguous. Can you use a `ForCausalLM` or `ForMaskedLM` instead?

Also maybe give examples in speech and vision (like `ForObjectClassification` or `ForCTC`.